docs/FAQ.md

```diff
@@ -5,17 +5,76 @@ nav_order: 11
 ---
 # Frequently Asked Questions
 
-### Why does `explain()` show that the GPU will be used even after setting `spark.rapids.sql.enabled` to `false`?
+* TOC
+{:toc}
 
-Apache Spark caches what is used to build the output of the `explain()` function. That cache has no
-knowledge about configs, so it may return results that are not up to date with the current config
-settings. This is true of all configs in Spark. If you changed
-`spark.sql.autoBroadcastJoinThreshold` after running `explain()` on a `DataFrame`, the resulting
-query would not change to reflect that config and still show a `SortMergeJoin` even though the new
-config might have changed to be a `BroadcastHashJoin` instead. When actually running something like
-with `collect`, `show` or `write` a new `DataFrame` is constructed causing Spark to re-plan the
-query. This is why `spark.rapids.sql.enabled` is still respected when running, even if explain shows
-stale results.
+### What versions of Apache Spark does the RAPIDS Accelerator for Apache Spark support?
+
+The RAPIDS Accelerator for Apache Spark requires version 3.0.0 or 3.0.1 of Apache Spark. Because the
+plugin replaces parts of the physical plan that Apache Spark considers to be internal the code for
+those plans can change even between bug fix releases. As a part of our process, we try to stay on
+top of these changes and release updates as quickly as possible.
+
+### Which distributions are supported?
+
+The RAPIDS Accelerator for Apache Spark officially supports [Apache
+Spark](get-started/getting-started-on-prem.md), [AWS EMR
+6.2.0](get-started/getting-started-aws-emr.md), [Databricks Runtime
+7.3](get-started/getting-started-databricks.md) and [Google Cloud
+Dataproc](get-started/getting-started-gcp.md).  Most distributions based off of Apache Spark 3.0.0
+should work, but because the plugin replaces parts of the physical plan that Apache Spark considers
+to be internal the code for those plans can change from one distribution to another. We are working
+with most cloud service providers to set up testing and validation on their distributions.
+
+### What CUDA versions are supported?
+
+CUDA 10.1, 10.2 and 11.0 are currently supported, but you need to download the cudf jar that 
+corresponds to the version you are using. Please look [here](download.md) for download 
+links for the latest release.
+
+### What is the right hardware setup to run GPU accelerated Spark?
+
+Reference architectures should be available around Q1 2021.
```

Comment:
It is Q1 2020. Do we have an update on this?

Reply:
No update yet, we are working on the hardware architectures and those will be out late Q1 or early Q2.